spec.md

```diff
@@ -147,10 +147,19 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
 | metadata | JSON object | An opaque object of metadata for a service plan. Controller treats this as a blob. Note that there are [conventions](https://docs.cloudfoundry.org/services/catalog-metadata.html) in existing brokers and controllers for fields that aid in the display of catalog data. |
 | free | boolean | When false, instances of this plan have a cost. The default is true |
 | bindable | boolean | Specifies whether instances of the service plan can be bound to applications. This field is OPTIONAL. If specified, this takes precedence over the `bindable` attribute of the service. If not specified, the default is derived from the service. |
+| [schemas](#schemas-object) | object | Schema definitions for service instances and bindings for the plan. |
 
 
 \* Fields with an asterisk are REQUIRED.
 
+##### Schemas Object
+
+| Response field | Type | Description |
+| --- | --- | --- |
+| service-instance | JSON schema object | The schema definition for a service instance. Schema definitions MUST be valid [JSON Schema draft v4](http://json-schema.org/). Broker authors SHOULD explicitly mention the JSON schema draft 4 version, as this will prevent breaking changes when in the future the OSB spec starts supporting future versions of JSON schema with low readability impact. Input parameters that can be used to create a service instance can be configured using a top-level `parameters` object. A service that does not support any input parameters for service instances SHOULD provide the following schema: `{ "$schema": "http://json-schema.org/draft-04/schema#", "additionalProperties": false }`. |
```

Comment:
The field names used here and in the example below are inconsistent.  The table uses the names `service-instance` and `service-binding`, while the examples use `service_instances` and `service_bindings`.  The field names used in the example are closer to the naming conventions used elsewhere in the spec.  Additionally, my understanding of this proposal was that we would have separate schemas for create and update of instances.  Did I understand incorrectly?